Formula/swi-prolog.rb

```diff
@@ -69,7 +69,7 @@ def install
     # './prepare' prompts the user to build documentation
     # (which requires other modules). '3' is the option
     # to ignore documentation.
-    system "echo '3' | ./prepare" if build.head?
+    system "echo", "'3' | ./prepare" if build.head?
```

Comment:
It looks like system calls with `|` in them may supposed to be exempt [already](https://github.com/Homebrew/brew/blob/a0d008af77785071ffba27dd995e5c794042f649/Library/Homebrew/cmd/audit.rb#L908).